Directory.Build.props

```diff
@@ -1,11 +1,11 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
     <LangVersion>9.0</LangVersion>
-    <Nullable>enable</Nullable>
+    <Nullable>disable</Nullable>
```

Comment:
Can we keep null checking?

Reply:
I would love to, but I spent ~10 minutes trying to add the proper annotations and (a) I didn't feel like I was adding value, and (b) I did not have any end in sight. This code makes *heavy* and intentional use of `null` values, and slathering a `?` on everything didn't feel worth the delay in merging this PR.

I've created a task to turn it back on after this PR merges if you'd like to take a crack at it: https://github.com/Faithlife/FaithlifeTesting/issues/24